src/containers/AssessmentContainer.ts

```diff
@@ -0,0 +1,29 @@
+import { connect, MapDispatchToProps, MapStateToProps } from 'react-redux'
+import { bindActionCreators, Dispatch } from 'redux'
+
+import { fetchAssessment } from '../actions/session'
+import Assessment, { DispatchProps, StateProps } from '../components/Assessment'
+import { IState } from '../reducers/states'
+
+export type OwnProps = { missionId: number }
+const mapStateToProps: MapStateToProps<StateProps, OwnProps, IState> = (state, props) => {
+  return {
+    missionId: props.missionId,
```

Comment:
why not `...props` so we don't have to touch this when we include more stuff in own props later on

Reply:
Addressed in #104